Source/Core/DolphinQt/DiscordHandler.cpp

```diff
@@ -0,0 +1,81 @@
+// Copyright 2018 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#ifdef USE_DISCORD_PRESENCE
+
+#include <iterator>
+
+#include <QApplication>
+
+#include "DolphinQt/DiscordHandler.h"
```

Comment:
This include should be first, for style reasons, since it's the matching header for this cpp file.